libs/RobotKit/include/RobotController.h

```diff
@@ -345,7 +345,7 @@ class RobotController : public interface::RobotController
 		auto is_autonomous_mode		= state_machine.is(system::robot::sm::state::autonomous_activities);
 		auto NOT_is_autonomous_mode = !is_autonomous_mode;
 
-		if (card == MagicCard::emergency_stop) {
+		if (card == MagicCard::emergency_stop && rtos::Kernel::Clock::now() - kSystemStartupTimestamp > 10s) {
```

Comment:
You don't want a parameter as the startup duration to be drown in the code, I suggest to set at the start of the method or set in member variables.